### ISSUE TITLE
Missing deprecation guide for session events

### ISSUE THREAD

**Issue by Gaurav0** [mainmatter/ember-simple-auth#2511]
Please explain how to migrate away from using session events to call sessionAuthenticated and sessionInvalidated.

ember-simple-auth.events.session-service

**Comment by mastastealth** [comment]
Yes please. I'm trying to cleanup the deprecations in an app here and am not seeing my extended session's `handleAuthentication` or `handleInvalidation`  ever being called, while the older `on` events will fire as expected.
